R/002_plot_components.R

```diff
@@ -2,7 +2,18 @@
 #
 # Contains functions `create_rawplot_components`, `create_deltaplot_components` and `create_violinplot_components`.
 
-# Function for creation of list of TRUE/FALSE for raw plot components that will be built
+#' Generates list of TRUE/FALSE for raw plot components that will be built
+#' 
+#' This function generates a list of booleans to be passed into the raw plot function.
```

Comment:
Can you reword this to `generates a list of booleans determining whether certain plot components will be constructed for the rawplot.`

Reply:
Done

Reply:
Oh oops, i didnt realise there were multiple lines with this: theres line 7 also

Thanks

Reply:
Done